config/routes.rb

```diff
@@ -117,8 +117,13 @@
     resources :webhook_endpoints, only: :index
     resource :config
     resources :badges, only: %i[index edit update new create]
-    get "/badge_achievements/award_badges", to: "badges#award"
-    post "/badge_achievements/award_badges", to: "badges#award_badges"
+    # These redirects serve as a safegaurd to prevent 404s for any Admins
+    # who have the old badge_achievement URLs bookmarked.
+    get "/badges/badge_achievements", to: redirect("/admin/badge_achievements")
```

Comment:
🔥  thank you for saving us all from ourselves 😉

Reply:
Any time! 😄